x-pack/plugins/siem/public/resolver/view/panel.tsx

```diff
@@ -47,6 +58,85 @@ export const Panel = memo(function Event({ className }: { className?: string })
     event: ResolverEvent;
   }
 
+  const history = useHistory();
+  const urlSearch = location.search;
+  const queryParams: {readonly crumbId: string, readonly crumbEvent: string} = useMemo(() => { 
+    return Object.assign({crumbId: '', crumbEvent: ''}, querystring.parse(urlSearch.slice(1)))},
+  [urlSearch]);
+  
+  const graphableProcesses = useSelector(dataSelectors.graphableProcesses);
+  const graphableProcessEntityIds = useSelector(dataSelectors.graphableProcessesEntityIds);
+  const relatedEvents = useSelector(selectors.relatedEvents);
+  /**
+   * Determine which set of breadcrumbs to display based on the query parameters
+   * for the table & breadcrumb nav
+   */
+  const whichTableViewAndBreadcrumbsToRender = useMemo(()=>{
+    const {crumbId, crumbEvent} = queryParams;
+    if(graphableProcessEntityIds.has(crumbEvent)){
```

Comment:
shouldn't this be crumbEventType? If so, why would that ever be in the graphableProcessEntityIds data structure? From my understanding this variable was going to be an enum of the ecs event categories + 'all' or w/e

Reply:
It was the last row in the table we talked about (b/c we can't know before we issue the request if it's _even_ a related event or not... at least until we get the stuff @jonathan-buttner was working on to inline that with the tree)

Crumb Title | crumbId | crumbEventType
-- | -- | --
Event List | null | null
\REGISTRY_USER\S1-5... | entity_id of REGISTRY event | entity_id of EXCEL process

Reply:
@kqualters-elastic :moneybag: SOLD :moneybag: on the call with @jonathan-buttner @michaelolo24